env/src/main/java/com/sap/cloud/security/config/Environments.java

```diff
@@ -5,6 +5,7 @@
  */
 package com.sap.cloud.security.config;
 
+
 import com.sap.cloud.security.config.cf.CFEnvironment;
```

Comment:
Did you apply formatter?

Reply:
yes, finally I've applied the formatting rules, should be gone 👍